h/buildext.py

```diff
@@ -149,6 +167,8 @@ def get_env(config_uri, base_url):
     env = paster.bootstrap(config_uri, request)
     request.root = env['root']
 
+    request.sentry = raven.Client(release=raven.fetch_package_version('h'))
```

Comment:
It isn't essential, but it is consistent with how we create the client in `sentry.py`.